### PR TITLE
Update keyboard movement

### PR DESCRIPTION
**Pull request by ldicker83**
Changes keyboard bindings so that movement in any direction is more intuitive.

### CODE REVIEW

File: OPHD/Common.h [hunk 1]
```diff
@@ -46,7 +46,11 @@ enum class Direction
 	East,
 	West,
 	North,
-	South
+	South,
+    NorthWest,
+    NorthEast,
+    SouthWest,
+    SouthEast
```
Comment (DanRStevens) [inline OPHD/Common.h:53]: Looks like spaces were used to indent here.

File: OPHD/States/MapViewState.cpp [hunk 2]
```diff
@@ -405,22 +405,22 @@ void MapViewState::onKeyDown(NAS2D::EventHandler::KeyCode key, NAS2D::EventHandl
 	{
 		case NAS2D::EventHandler::KeyCode::KEY_w:
 		case NAS2D::EventHandler::KeyCode::KEY_UP:
-			mMapView->moveView(Direction::North);
+			mMapView->moveView(Direction::NorthWest);
```
Comment (DanRStevens) [inline OPHD/States/MapViewState.cpp:408]: You know, if we updated the parameter type here, we could change this from:
```cpp
mMapView->moveView(Direction::NorthWest);
```
To:
```cpp
mMapView->moveView(DirectionNorthWest);
```

Same readability, but without needing an enum to offset lookup.

**Review by DanRStevens — COMMENTED** [review]
Yes! This is so much more intuitive. Oddly, I wouldn't have really thought to move diagonally like that, so that it appears to move straight. This is so much better.

File: OPHD/States/MapViewState.cpp (hunk 2 above)
Comment (DanRStevens) [reply OPHD/States/MapViewState.cpp:408]: Ahh, but there's also the up/down buttons handled by that function.

**Comment by ldicker83** [comment]
After not having played the game for awhile it threw me off a bit and then I remembered you talking about how it wasn't very intuitive. So I figured I'd modify it a bit :D Means it ultimately moves '2 tiles' versus just one but it seems to make more sense. Since we both had that experience others must have as well.

File: OPHD/Common.h (hunk 1 above)
Comment (ldicker83) [reply OPHD/Common.h:53]: Ooof, yeah, that's a change in VS2022's latest things, I've been using the spaces over tabs lately in new projects. Will have to update the text editor to keep this consistent.

File: OPHD/States/MapViewState.cpp (hunk 2 above)
Comment (ldicker83) [reply OPHD/States/MapViewState.cpp:408]: True, but I kinda like the indirection a bit. Kinda like namespaces but without the namespace

**Pull request merged**